docs/databases/hadoop/how-to-install-and-set-up-hadoop-cluster/index.md

```diff
@@ -146,7 +150,7 @@ export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/jre
 
 ### Set NameNode Location
 
-On each node update `~/hadoop/etc/hadoop/core-site.xml` you want to set the NameNode location to **node-master** on port `9000`:
+Update your `~/hadoop/etc/hadoop/core-site.xml` file to set the NameNode location to **node-master** on port `9000`:
```

Comment:
We don't need to do this on each node since it is already performed in a later step.